.github/workflows/open-develop-pr.yaml

```diff
@@ -0,0 +1,35 @@
+##
+## Auto-opens a PR from main -> develop after a release has been published.
+##
+
+name: Open Develop PR
+
+on:
+  release:
+    types:
+      - released
+  workflow_dispatch:
+
+jobs:
+  run:
+    name: Upsteam develop branch
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Clarinet
```

Comment:
```suggestion
      - name: Checkout clarity-repl
```

Reply:
Fixed in 1fe39ff438e7da97171d6a7e4685854debaddea5.